lib/utils/cert-utils.js

```diff
@@ -0,0 +1,103 @@
+const os = require('os');
+const path = require('path');
+const fs = require('fs-extra');
+const forge = require('node-forge');
+const logger = require('@blackbaud/skyux-logger');
+
+const certCommonName = 'SKYUX-Developer-Certificate';
+const certName = 'skyux-server.pem';
+const keyName = 'skyux-server.key';
+
+const defaultCertRootPath = path.resolve(`${os.homedir()}/.skyux/certs/`);
+const defaultCertPath = path.join(defaultCertRootPath, certName);
+const defaultKeyPath = path.join(defaultCertRootPath, keyName);
+
+function error(msg) {
+  logger.error(msg);
+  return false;
+}
+
+function validate(argv) {
+  const cert = getCertPath(argv);
+  const key = getKeyPath(argv);
+
+  if (!fs.existsSync(cert)) {
+    return error(`Error locating certificate: ${cert}`);
+  }
+
+  if (!fs.existsSync(key)) {
+    return error(`Error locating key: ${key}`);
```

Comment:
If `skyux new` (or any other command) fails because the certificate is not found, would it be worth asking if the user ran `skyux certs trust`, just as a reminder? Something to the effect of...
```
`There was an error locating the certificate key \`${key}\`. Did you run \`skyux certs trust\``?
```

Same for the error above.

Reply:
I call out a very similar message in two places.

1. In this PR at https://github.com/blackbaud/skyux-sdk-cli/pull/36/files#diff-168726dbe96b3ce427e7fedce31bb0bcR188-R191 Although I could reword it to mention running `skyux certs trust` as you suggested.

2. In the Builder PR at https://github.com/blackbaud/skyux-sdk-builder/pull/168/files#diff-78cf2d7696a73e189749f3a595673fafR7-R10

Do you think those are sufficient?

Reply:
Yes, that will do! I think it would be helpful to explain the error and then what the user can do to solve it.

Reply:
Slightly updated the messaging for # 1.  Let me know if that doesn't resolve your suggestion.